pages/api/auth/[...auth0].js

```diff
@@ -0,0 +1,3 @@
+import { handleAuth } from '@auth0/nextjs-auth0';
+
+export default handleAuth();
```

Comment:
Kudos to myself, implementing with passwordless login gave a lot of issues. I had to use only the default login page 😘